isotovideo

```diff
@@ -172,12 +172,24 @@ sub signalhandler_chld {
 sub init_backend {
     my ($name) = @_;
     $bmwqemu::vars{BACKEND} ||= "qemu";
-    my $dir = getcwd;
-    # make sure the needles are initialized before the backend process is started
     no autodie qw(system);
-    my $res = system("rsync -avP --delete --exclude=.git $bmwqemu::vars{PRODUCTDIR}/needles/ $dir/needles/");
-    die "Failed to rsync needles: $@" if $res;
-    needle::init("$dir/needles");
+
+    #We want to use the pool directory instead of the
+    # cwd (which would be the worker's directory)
+    # to be able to share the needles between workers
+    my $shared_needle_dir = getcwd() . "/..";
```

Comment:
1) Configurable cache dir, 2) don't forget that worker can run tests from different distri - I suggest to add distri to the cache dir and 3) there is the plan worker can be shared between more webuis - add openqa host to the path.

So at the end path should IMO be something like `File::Spec->catdir($cachedir_from_config, $openqa_host, $distri)`.